src/components/FilterableList/ContainerFilterableList.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+import FilterableList from './FilterableList';
+import { getCurrentCityData, SET_CURRENT_CITY_NAME } from '../../ReduxStore/actions/currentCity';
+
+const MapDispatchToProps = dispatch => {
+  return {
+    getCurrentCityData: id => dispatch(getCurrentCityData(id)),
+    setCurrentCityName: name => dispatch({ type: SET_CURRENT_CITY_NAME, payload: name })
```

Comment:
Can you elaborate why one action is made with action creator and another is not? Could you please make them both with action creators